### ISSUE TITLE
npm WARN config production Use `--omit=dev` instead.

### ISSUE THREAD

**Issue by matthew-white** [getodk/central#357]
When I build Central, I see the following warning:

```
Step 14/22 : RUN npm clean-install --production --legacy-peer-deps
 ---> Running in 6203f5e5415a
npm WARN config production Use `--omit=dev` instead.
```

Should we be doing something different? Is there a way for us to address this warning so that it's not shown during the build?

**Comment by alxndrsn** [comment]
@matthew-white what command are you using to build central?  Which branch/commit are you on?

**Comment by matthew-white** [comment]
Hello!

> what command are you using to build central?

`docker-compose build`

> Which branch/commit are you on?

`next`

**Comment by alxndrsn** [comment]
Also re-creatable with `docker-compose build service`.

**Comment by alxndrsn** [comment]
> **`production`**
>
> * Default: null
> * Type: null or Boolean
> * DEPRECATED: Use `--omit=dev` instead.
>
> Alias for `--omit=dev`
>
> -- _https://docs.npmjs.com/cli/v9/using-npm/config#production_

It's official!  I'll open a PR to fix.

**Comment by matthew-white** [comment]
Closed by #358.
